src/wombats/game/initializers.clj

```diff
@@ -127,38 +120,44 @@
                                                   (assoc-in zakano-state [:state :code] code)))
                                          {} zakano)))))
 
-(defn- is-start-of-round?
+(defn initialize-game-state
+  "Prepares the raw built up game state for the frame processor by adding required
+  information to context."
   [game-state]
-  (nil? (get-in game-state [:frame :frame/round-start-time] nil)))
-
-(defn- sleep-round
-  [game-state]
-  (when (> (get-in game-state [:frame :frame/round-number]) 1)
-    (Thread/sleep (get-in game-state [:game-config :game/round-intermission]))))
+  (-> game-state
+      (add-players-to-game)
+      (set-initiative-order)
+      (set-zakano-state)
+      (source-player-code)
+      (source-zakano-code)))
 
 (defn- set-round-start-time
+  "Sets the round start time"
   [game-state]
   (assoc-in game-state [:frame :frame/round-start-time] (->> (t/now)
                                                              (format "#inst \"%s\"")
                                                              (read-string))))
 
 (defn- set-round-status
+  "Sets the round status"
   [game-state]
   (assoc-in game-state [:game-config :game/status] :active))
 
-
 (defn initialize-round
+  "Adds round metadata to game state"
   [game-state]
   (-> game-state
       (set-round-start-time)
-      (set-round-status)
-      (add-players-to-game)
-      (set-initiative-order)
-      (set-zakano-state)
-      (source-player-code)
-      (source-zakano-code)))
+      (set-round-status)))
+
+(defn- update-initiative-order
+  "Rotates the initiative order"
```

Comment:
This one in particular wouldn't be too useful since initiative order is not retuned in the Lambda fn